packages/ckeditor5-core/package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@ckeditor/ckeditor5-engine": "41.4.2",
     "@ckeditor/ckeditor5-utils": "41.4.2",
+    "@ckeditor/ckeditor5-watchdog": "41.4.2",
```

Comment:
Do we have to add watchdog to core DDL package? @pomek :thinking:

Reply:
I think we should.

Reply:
It's already there, my bad 🥲